README.md

```diff
@@ -1 +1,85 @@
-hl-be-website
+# NEOLOL Back-end
+
+#### [Front-End Repository](https://github.com/Neolol-source/Neolol-FE)  
+
+#### [Feature list on Trello](https://trello.com/b/b5gN1DpV/neolol-planned-features)  
+
+
+## Installation
+
+Install [MySQL](https://dev.mysql.com/downloads/mysql/)  
+```bash
+$ sudo apt update
+$ sudo apt install mysql-server
+$ sudo systemctl restart mysql
+```  
+
+Configure MySQL credentials and create a Database 
+```bash
+$ mysql -u root -p
+```  
+```sql
+ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '[PASSWORD]';
+FLUSH PRIVILEGES;
+CREATE DATABASE test;
+```
+
+## Usage  
+
+### Running this Back-End
+On the repository folder, install all dependencies.  
+```bash
+$ npm install
+```   
+
+Make a copy of the file `.env-TEMPLATE` and name it `.env`;  
+Fill the necessary information following the comments.  
+
+Run the server.  
+```bash
+$ node server.js
+```  
```

Comment:
Better to use the npm scripts inside package.json for an easier experience

Reply:
Done, and added `npm run sample` and `npm run clean` as well